docs/content/s3.md

```diff
@@ -1920,6 +1923,138 @@ d) Delete this remote
 y/e/d> y
 ```
 
+### Tencent COS {#tencent-cos}
+
+[Tencent Cloud Object Storage (COS)](https://intl.cloud.tencent.com/product/cos) is a distributed storage service offered by Tencent Cloud for unstructured data. It is secure, stable, massive, convenient, low-delay and low-cost.
+
+To configure access to Tencent COS, follow the steps below:
+
+1. Run `rclone config` and select `n` for a new remote.
```

Comment:
Do you think the config walkthrough is necessary here? It is almost the same as the main config - maybe just tell the user that they **must** choose provider `TencentCOS`?

Reply:
Yes, it's very similar with the main config. But it has different endpoints, different Storage Class and ACL configurations, compare to s3. I still prefer that adding a standalone config walkthrough is more clear for the users, and easy to use.